docs/getting-started/download.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: docs
+title: Getting Started
+subtitle: An overview of Athena, including how to download and use it, some basic templates and examples, and more.
+---
+
+# Download
+The Athena Framework is avaailable to download in a few ways, depeneding on your needs.
+
+<a href="https://github.com/UCF/Athena-Framework/archive/v1.0.0-alpha.zip">Download Athena Framework v1.0.0-alpha</a> <span class="badge badge-primary">alpha</span>
+
+<hr>
+
+# Contents
+The un-zipped folder will contain minified files as well as source code (Athena is open-source).
```

Comment:
Maybe add an extra line here explaining when to use minified files vs source code--e.g. "Projects that need to extend Athena and/or reference Athena's Sass variables should utilize the source code files in /src/.  Projects that just need to include Athena without modifications can use the already-minified files in /dist/.  See the Installation instructions (link) for more details"